xbmc/utils/test/TestCryptThreading.cpp

```diff
@@ -15,3 +22,58 @@ TEST(TestCryptThreadingInitializer, General)
   std::cout << "g_cryptThreadingInitializer address: " <<
     testing::PrintToString(&g_cryptThreadingInitializer) << "\n";
 }
+
+#define PVTID_NUM_THREADS 10
+
+TEST(TestCryptThreadingInitializer, ProducesValidThreadIds)
+{
+  std::thread testThreads[PVTID_NUM_THREADS];
+
+  std::vector<unsigned long> gatheredIds;
+  CCriticalSection gatheredIdsMutex;
+
+  std::atomic<unsigned long> threadsWaiting{0};
+  std::atomic<bool> gate{false};
+
+  for (int i = 0; i < PVTID_NUM_THREADS; i++)
+  {
+    testThreads[i] = std::thread([&gatheredIds, &gatheredIdsMutex, &threadsWaiting, &gate]() {
+        threadsWaiting++;
+
+        while (!gate);
```

Comment:
clang-formats reformats this to the following.
```
      while (!gate)
        ;
```

Reply:
I'm not sure what you're asking me to do. Personally I think you should fix your clang-formats configuration.

Reply:
If you want to keep it like this, it's okay. Just wanted to mention that clang-format would format it in a different way.

Reply:
Okay. I think it looks cleaner this way but that's probably just because I'm used to seeing it that way.